setup.cfg

```diff
@@ -70,7 +70,8 @@ addopts = --doctest-rst
 
 [flake8]
 max-line-length = 100
-select = E101,W191,W291,W292,W293,W391,E111,E112,E113,E502,E722,E901,E902
+# E402: Module level import not at top of file
+ignore = E402
```

Comment:
@ibusko , currently it only checks for a few via `select`. What I did here is to have it check for everything except `ignore`. Does that clear up your confusion?